demo/platformer/end_menu.py

```diff
@@ -30,4 +30,4 @@ def on_switch():
 
 
 scene.on_switch = on_switch
-scene.add_ui(restart_button, screenshot_button, time_text)
+scene.add(restart_button, screenshot_button, time_text)
```

Comment:
Same as in discord: 
ok just an idea. Since this is a different behavious now (they will move w/ camera b/c we didn't ignore_cam them). We should still have an add_ui, that will just add normally and turn on the ignore_cam or we should turn on the ignore_cam for these in the demos

Reply:
n o i dont want multiple functions